parts/kubernetesagentcustomdata.yml

```diff
@@ -107,7 +107,7 @@ write_files:
     DOCKER_OPTS=
     CUSTOM_CMD=/bin/true
     KUBELET_REGISTER_SCHEDULABLE=true
-    KUBELET_NODE_LABELS=role=agent
+    KUBELET_NODE_LABELS=role=agent{{ GetKubernetesCustomLabels . }}
     KUBELET_POD_INFRA_CONTAINER_IMAGE={{WrapAsVariable "kubernetesPodInfraContainerSpec"}}
```

Comment:
👍  There wasn't a real reason for not including `role=agent` in `GetKubernetesCustomLabels`. I'll implement your suggestion, however!